src/test/cpp/util/transformer.cpp

```diff
@@ -116,14 +116,25 @@ void Transformer::createSedCommandFile(const std::string& regexName,
 
 	std::string tmp;
 
+	auto sedsaniziter = [] (const std::string &in, const std::string &sedseperator = "Q") {
+		std::string ret = in;
+		std::string replace_to = "\\" + sedseperator;
+		size_t pos = 0;
+		while((pos = ret.find(sedseperator, pos)) != std::string::npos) {
```

Comment:
Please add a newline before and after the `while`, make things better readable.

Reply:
should be fixed now by 38729af.